rpc/api.proto

```diff
@@ -1168,3 +1169,14 @@ message GetRawBlockRequest {
 message GetRawBlockResponse {
 	bytes block = 1;
 }
+
+message GetCoinjoinOutputspByAcctRequest {}
+
+message coinjoinTxsSumByAcct {
+	uint32 account_number = 1;
+	int32 coinjoin_txs_sum = 2;
+}
+
+message GetCoinjoinOutputspByAcctResponse {
+	repeated coinjoinTxsSumByAcct data = 1;
+}
```

Comment:
there should be a newline on the last line.  sometimes editors (windows ones in particular) won't add this for some reason.